packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-clickable-description.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import JurisdictionClickableDescription from '../jurisdiction-clickable-description';
+
+describe('JurisdictionClickableDescription', () => {
+    const mock_props = {
+        clickable_description: [
+            {
+                text: 'Click here',
+                type: 'link' as const,
+            },
+            {
+                text: 'to learn more about the documents required for verification.',
+                type: 'text' as const,
+            },
+        ],
+        toggleCardFlip: jest.fn(),
+    };
+
+    it('should render JurisdictionClickableDescription', () => {
+        render(<JurisdictionClickableDescription {...mock_props} />);
+        expect(screen.getByText('Click here')).toBeInTheDocument();
+        expect(screen.getByText('to learn more about the documents required for verification.')).toBeInTheDocument();
```

Comment:
Click here will be displayed in red text. could you please check for the presence of class `.cfd-card-clickable-description-link`?
<img width="306" alt="Screenshot 2023-04-26 at 4 03 33 PM" src="https://user-images.githubusercontent.com/122449658/234569137-5b3cd49f-d50c-4328-b41a-cc26c0d9b7f7.png">